server/api.js

```diff
@@ -139,26 +139,63 @@ router.get("/buyer-tender", async (req, res) => {
 	const buyerId = 1;
 	let page = parseInt(req.query.page) || 1;
 	const itemsPerPage = 25;
-
 	const offset = (page - 1) * itemsPerPage;
+	const totalBuyerTenders = await db.query("SELECT COUNT(buyer_id) FROM bid WHERE buyer_id = $1", [buyerId]);
+	const totalPages = Math.ceil(totalBuyerTenders.rows[0].count / itemsPerPage);
 	const result = await db.query("SELECT * FROM tender WHERE buyer_id = $1 LIMIT $2 OFFSET $3", [buyerId, itemsPerPage, offset]);
+
 	result
-		? res.send(result.rows)
+		? res.send({
+			"result": result.rows, "pagination": {
+				"itemsPerPage": 25,
```

Comment:
```suggestion
				"itemsPerPage": itemsPerPage,
```

Or even better, use a constant that can be used on all api endpoints